lib/rpm.rb

```diff
@@ -41,10 +41,9 @@ def self.transaction(root = '/')
   # @param [String] name Name of the macro
   # @return [String] value of macro +name+
   def self.[](name)
-    val = ''
     buffer = ::FFI::MemoryPointer.new(:pointer, 1024)
     buffer.write_string("%{#{name}}")
-    ret = RPM::C.expandMacros(nil, nil, buffer, 1024)
```

Comment:
Can you actually leave this one in?  We should be doing error checking and I'm okay with it being "unused" for now so I don't forget about it.